custom-image-export.lke/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html class="no-js" lang="">
```

Comment:
We should create a separate folder for the assets instead of the *.lke itself directly